manifests/integreatly-rhsso/rhsso-8.0.1/keycloakoperator.8.0.1.clusterserviceversion.yaml

```diff
@@ -185,7 +185,7 @@ spec:
                             fieldPath: metadata.name
                       - name: OPERATOR_NAME
                         value: keycloak-operator
-                    image: quay.io/pb82/keycloak-operator:no-default-credentials
+                    image: quay.io/wei_lee/keycloak-operator:intly
```

Comment:
why is this happening here? Does the image from @wei-lee include the changes we need from @pb82

Reply:
Yes, Wei's image includes Peter's changes and also the changes from https://github.com/keycloak/keycloak-operator/pull/125, without which pretty much all integreatly monitoring is broken, because it breaks prometheus config.

Reply:
ok, cool